imagetagger/imagetagger/annotations/models.py

```diff
@@ -408,6 +407,17 @@ class VECTOR_TYPE():
     node_count = models.IntegerField(default=0)
     enable_concealed = models.BooleanField(default=True)
     enable_blurred = models.BooleanField(default=True)
+    """
```

Comment:
It might be a good idea to provide a method to get the rendered markdown in this model class.